src/components/govukComponents/BackLink.js

```diff
@@ -1,18 +1,30 @@
 import React from 'react'
-import Link from '../utils/Link.js'
+import PropTypes from 'prop-types'
+import Link from '../utils/Link'
 
 function BackLink(props) {
-  const contents = props.html ? props.html : props.text
+  const contents = props.html || props.text
 
-  return <Link classes={`govuk-back-link ${props.classes}`} attributes={props.attributes} href={props.href} to={props.to}>
-    {contents}
-  </Link>
+  return (
+    <Link classes={`govuk-back-link ${props.classes}`} attributes={props.attributes} href={props.href} to={props.to}>
+      {contents}
+    </Link>
+  )
 }
 
 BackLink.defaultProps = {
-  href: "/",
-  text: 'Back',
-  classes: ''
+  href: '/',
+  text: <>Back</>,
+  classes: '',
+}
+
+BackLink.propTypes = {
+  attributes: PropTypes.object,
+  classes: PropTypes.string,
+  href: PropTypes.string,
+  html: PropTypes.node,
+  text: PropTypes.node,
```

Comment:
As above

Reply:
The default value can be a string, and I have changed it, but PropTypes.node is there to catch all "renderable" things, not just strings. The contents could be jsx